### PR TITLE
RK-10209 - Minor update to source map copy

### PR DESCRIPTION
**Pull request by LiranLast**

### CODE REVIEW

File: docs/node-setup.md [hunk 1]
```diff
@@ -113,9 +113,9 @@ If you face cases where some variables are not collected, or where some breakpoi
 - [**Typescript**](https://www.typescriptlang.org/) - use the `--inlineSources` flag. For [**ts-node**](https://github.com/TypeStrong/ts-node) add source maps using the `tsconfig.json` [file](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
 - [**CoffeeScript**](https://coffeescript.org/) - use the `-M` or `-m` flags.
 
-#### TypeScript & Webpack
+#### Complex Transpiling Configurations
```
Comment (Rookeret73) [inline docs/node-setup.md:116]: Advanced, instead of Complex

File: docs/node-setup.md [hunk 2]
```diff
@@ -113,9 +113,9 @@ If you face cases where some variables are not collected, or where some breakpoi
...
 
-Because of the way Webpack handles TypeScript inputs, an additional step is needed:
+In some cases, if use multiple transpiling steps, such as running TypeScript followed by Webpack, you might need an additional step to make sure source maps end up in the right place.
```
Comment (Rookeret73) [inline docs/node-setup.md:118]: If multiple transpiling steps are used (for example - TypeScript followed by Webpack), you may need to add an additional step. The additional step should make sure the source maps end up in the right place.

//What is the right place? Does a reasonable user know how to add such a step?

Comment (LiranLast) [reply docs/node-setup.md:118]: The step is 1-2 below

**Review by Rookeret73 — APPROVED** [review]

**Pull request merged**